docs/kernel-policy.md

```diff
@@ -0,0 +1,85 @@
+# Firecracker's Kernel Support Policy
+
+Being a virtual machine monitor, Firecracker represents a component in a
+larger stack, one in which it is tightly coupled with the guest and host
+kernels on which it is run.
```

Comment:
I  would add that With this kernel support policy we want to offer customers predictability into future kernel related changes.